### PR TITLE
Remove template miner

### PR DESCRIPTION
**Pull request by Eugene-hu**
-Deprecate Template miner
-Adds priority for the core server

### CODE REVIEW

File: scripts/update.sh [hunk 1]
```diff
@@ -0,0 +1,232 @@
+
```
Comment (isabella618033) [inline scripts/update.sh]: why is this file here?

File: tests/unit_tests/bittensor_tests/test_neuron.py [hunk 2]
```diff
@@ -237,89 +237,6 @@ def exit_early(*args, **kwargs):
         # Should try to register the neuron
         mock_register.assert_called_once()
 
-def test_templateminer_reregister_flag_false_exit():
```
Comment (isabella618033) [inline tests/unit_tests/bittensor_tests/test_neuron.py:240]: i think this is sth vune added recently, and it checkes for registration, so maybe convert that for sth core_server?

Comment (isabella618033) [reply tests/unit_tests/bittensor_tests/test_neuron.py:240]: same for the tst below

**Review by isabella618033 — COMMENTED** [review]
LGTM, left you small comments xd

File: tests/unit_tests/bittensor_tests/test_neuron.py (hunk 2 above)
Comment (Eugene-hu) [reply tests/unit_tests/bittensor_tests/test_neuron.py:240]: Theres one for core_server and core_validator already. So I just removed the tests for template miner

File: scripts/update.sh (hunk 1 above)
Comment (Eugene-hu) [reply scripts/update.sh]: Oops nice catch, that shouldnt be there

File: bittensor/_neuron/text/core_server/run.py [hunk 3]
```diff
@@ -166,7 +186,7 @@ def time_check():
         try:
             registration_check()
 
-            stake_check()
+            #stake_check()
```
Comment (shibshib) [inline bittensor/_neuron/text/core_server/run.py:189]: if we don't need this let's just delete it entirely.

**Review by shibshib — APPROVED** [review]

File: bittensor/_neuron/text/core_server/run.py (hunk 3 above)
Comment (Eugene-hu) [reply bittensor/_neuron/text/core_server/run.py:189]: This is temporary; we will need to turn on these checks when we get to Nakamoto
